NTumbleBit/Configuration/RPCArgs.cs

```diff
@@ -83,7 +83,7 @@ public RPCClient ConfigureRPCClient(Network network)
 			try
 			{
 				var address = new Key().PubKey.GetAddress(network);
-				var isValid = ((JObject)rpcClient.SendCommand("validateaddress", address.ToString()).Result)["isvalid"].Value<bool>();
+				var isValid = rpcClient.ValidateAddress(address).IsValid;
```

Comment:
@DanGould if you fix the RPCClient on the FullNode side, this will be:

`var isValid = rpcClient.ValidateAddress(address);`

Reply:
@DanGould but the code here is already MUCH cleaner. Good job 👍